src/main/java/Truck.java

```diff
@@ -0,0 +1,9 @@
+class Truck extends Machine {
```

Comment:
Consider using an access modifier for the `Truck` class. Choosing the right access level for classes is important to ensure encapsulation. If this class is intended to be accessed throughout your application, `public` would be appropriate.